### PR TITLE
mysql-operator

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [x] Updated/Added any relevant [documentation] and [examples].
- [x] Unit/E2E Tests added.

### CODE REVIEW

File: experimental/addons/mysql-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+```shell
+vela addon enable mysql-operator
+```
+
+## uninstall
+
+```shell
+vela addon disable mysql-operator
+```
```
Comment (wonderflow) [inline experimental/addons/mysql-operator/README.md:15]: we need more user guide here

File: experimental/addons/mysql-operator/definitions/mysql-cluster.cue [hunk 2]
```diff
@@ -0,0 +1,68 @@
+"mysql-cluster-v1": {
```
Comment (wonderflow) [inline experimental/addons/mysql-operator/definitions/mysql-cluster.cue:1]: This is a name of the component type, not that good

File: experimental/addons/mysql-operator/definitions/pv.cue [hunk 3]
```diff
@@ -0,0 +1,39 @@
+pv: {
```
Comment (wonderflow) [inline experimental/addons/mysql-operator/definitions/pv.cue:1]: the name is too short and will conflict with others

File: experimental/addons/mysql-operator/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: mysql-operator
+version: 2.0.4
+description: A Kubernetes operator for Mysql Cluster
+url: https://github.com/mysql/mysql-operator
+
+tags:
+  - mysql/mysql-operator
+  - 8.0.29-2.0.4
```
Comment (wonderflow) [inline experimental/addons/mysql-operator/metadata.yaml:8]: we should give more userful information in tags, it's confusing by `8.0.29-2.0.4`, a magic number

**Comment by oeular** [comment]
@wonderflow  Thanks for your review. I update the related files.

**Review by oeular — COMMENTED** [review]
Done

**Comment by wangyikewxgm** [comment]
> @wangyikewxgm I share a demo in README.md, it's clear and easy-to-go.

IMO the readme isn't a good place to show the demo, it will be showed in velaux addon's page, too large readme will bring a  bad user-experience. In fact we want put all addon usage in example dir. So I suggest move the demo there.

**Comment by oeular** [comment]
@wangyikewxgm  In fact ,as a user, the first wanted-thing is to show him directly how to get started as soon as quickly, and at the best, less docs are needed to search.

**Comment by wangyikewxgm** [comment]
> @wangyikewxgm In fact ,as a user, the first wanted-thing is to show him directly how to get started as soon as quickly, and at the best, less docs are needed to search.

The meaning of the `example dir` is that we can put more and more demos  in  for every addon in the future, a quick easy demo in readme is just fine.

File: experimental/addons/mysql-operator/resources/parameter.cue [hunk 5]
```diff
@@ -0,0 +1,2 @@
+parameter: {
+}
```
Comment (wangyikewxgm) [inline experimental/addons/mysql-operator/resources/parameter.cue:2]: why need an empty parameter file?

**Review by wangyikewxgm — APPROVED** [review]

**Pull request merged**